numba_dppy/examples/debug/commands/break_file:func

```diff
@@ -0,0 +1,17 @@
+# Run: gdb-oneapi -q -command commands/break_func:func python
+set trace-commands on
+set pagination off
+set breakpoint pending on
+break simple_sum.py:data_parallel_sum
+run simple_sum.py
+# Expected:
+# Thread 2.2 hit Breakpoint 1, with SIMD lanes [0-7], __main__::data_parallel_sum () at simple_sum.py:20
+# 20      @dppy.kernel(debug=True)
+continue
+# Expected:
+# Thread 2.3 hit Breakpoint 1, with SIMD lanes [0-1], __main__::data_parallel_sum () at simple_sum.py:20
+# 20      @dppy.kernel(debug=True)
+continue
+# Expected:
+# Done
```

Comment:
Miss echo Done\n

Reply:
Text "Done" appears in terminal after second continue command because debugger runs through full script.